README.md

```diff
@@ -151,6 +151,73 @@ To stop Explorer after it has been started with `PM2`, issue the following comma
 
 `pm2 stop lisk-explorer`
 
+## Docker
+
+### Using docker-compose
+
+Update `docker-lisk-core.env` to choose your preferred node. You can easily switch between Mainnet and Testnet nodes by changing content of the env file.
+
+#### Starting application
+
+To start Explorer type the following command:
+
+```
+docker-compose up
```

Comment:
Almost everyone will want to use `docker-compose up -d` I think; we should at least mention it.